src/cc/Httpd.cpp

```diff
@@ -43,19 +45,54 @@ bool Httpd::start()
         return false;
     }
 
-    m_daemon = MHD_start_daemon(MHD_USE_SELECT_INTERNALLY, static_cast<uint16_t>(m_options->ccPort()), nullptr, nullptr, &Httpd::handler,
-                                this, MHD_OPTION_CONNECTION_TIMEOUT, (unsigned int) 10, MHD_OPTION_END);
+#   ifndef XMRIG_NO_TLS
```

Comment:
Pfui!

Reply:
It's only to avoid ssl dependencies for non-tls compilation